src/tensors/levels/continuouslevels.jl

```diff
@@ -0,0 +1,314 @@
+                    stop = (ctx, ext) -> value(my_i_end),
+                    body = (ctx, ext) -> Stepper(
+                        seek = (ctx, ext) -> quote
+                            if $(lvl.ex).right[$my_q] < $(ctx(getstart(ext)))
+                                $my_q = scansearch($(lvl.ex).right, $(ctx(getstart(ext))), $my_q, $my_q_stop - 1)
+                            end
+                        end,
+                        body = Thunk(
+                            preamble = quote
+                                $my_i_start = $(lvl.ex).left[$my_q]
+                                $my_i_stop = $(lvl.ex).right[$my_q]
+                            end,
+                            body = (ctx) -> Phase(
+                                stop = (ctx, ext) -> value(my_i_stop),
+                                body = (ctx, ext) -> Thunk( 
+                                     body = (ctx) -> Pipeline([
+                                        Phase(
+                                            stop = (ctx, ext) -> call(-, value(my_i_start), 1),
+                                            body = (ctx, ext) -> Run(Fill(virtual_level_default(lvl))),
+                                        ),
+                                        Phase(
+                                            body = (ctx,ext) -> Run(
+                                                                    body = (get_reader(VirtualSubFiber(lvl.lvl, value(my_q)), ctx, protos...))
+                                                                   )
+                                        )
+                                    ]),
+                                    epilogue = quote
+                                        $my_q += ($(ctx(getstop(ext))) == $my_i_stop)
+                                    end
+                                )
+                            )
+                        )
+                    )
+                ),
+                Phase(
+                    body = (ctx, ext) -> Run(Fill(virtual_level_default(lvl)))
+                )
+            ])
+        )
+    )
+end
+
+
+is_laminable_updater(lvl::VirtualContinuousLevel, ctx, ::Union{Nothing, Extrude}) = false
+get_updater(fbr::VirtualSubFiber{VirtualContinuousLevel}, ctx, protos...) = 
+    get_updater(VirtualTrackedSubFiber(fbr.lvl, fbr.pos, ctx.freshen(:null)), ctx, protos...)
+
+function get_updater(fbr::VirtualTrackedSubFiber{VirtualContinuousLevel}, ctx, ::Union{Nothing, Extrude}, protos...)
+    (lvl, pos) = (fbr.lvl, fbr.pos) # Jaeyeon : Accessing with the position obtained from the parent fiber
+    tag = lvl.ex
+    Tp = lvl.Tp
+    Ti = lvl.Ti
+    qos = ctx.freshen(tag, :_qos)
+    qos_fill = lvl.qos_fill
+    qos_stop = lvl.qos_stop
+    dirty = ctx.freshen(tag, :dirty)
+    
+    Furlable(
+        tight = lvl, # Jaeyeon : Something related to error handling in unfurl.jl
+        size = virtual_level_size(lvl, ctx), # Jaeyeon : list of all extents of subfiber's shape? 
```

Comment:
This is right. Also, I'll be removing the size field on furlable in subsequent versions of the code.